sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -783,7 +784,7 @@ __ESIMD_API
                __ESIMD_NS::simd_mask<N> pred = 1) {
 #ifdef __ESIMD_FORCE_STATELESS_MEM
   return lsc_gather<T, NElts, DS, L1H, L2H>(
-      reinterpret_cast<T *>(acc.get_pointer().get()), offsets, pred);
+      __ESIMD_DNS::accessorToPointer<T>(acc), offsets, pred);
```

Comment:
I suppose this fix address some particular problem/regression.
We'll need to apply it to couple more places in ESIMD , e.g. https://github.com/intel/llvm/blob/229246b1a02f52b6368f35981c6936fd16dc3afa/sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp#L862

Reply:
Fixed in 766883e676fba04908ff80892891b51decd5764a

Note that this piece of code seems to be untested